src/planwise/client/projects2/components/settings.cljs

```diff
@@ -36,66 +36,84 @@
 (defn- current-project-input
   [label path transform]
   (let [current-project (rf/subscribe [:projects2/current-project])
-        value           (or (get-in @current-project path) "")
-        change-fn       #(rf/dispatch-sync [:projects2/save-key path (-> % .-target .-value transform)])]
-    [m/TextField {:type "text"
-                  :label label
+        value (or (get-in @current-project path) "")
+        change-fn #(rf/dispatch-sync [:projects2/save-key path (-> % .-target .-value transform)])]
+    [m/TextField {:type      "text"
+                  :label     label
                   :on-change change-fn
-                  :value value}]))
+                  :value     value}]))
 
 (defn- project-start-button
   [_ project]
-  [m/Button {:id "start-project"
-             :type "button"
-             :on-click (utils/prevent-default #(dispatch [:projects2/start-project (:id project)]))}
+  [m/Button {:id         "start-project"
+             :type       "button"
+             :unelevated "unelevated"
+             :on-click   (utils/prevent-default #(dispatch [:projects2/start-project (:id project)]))}
    (if (= (keyword (:state project)) :started) "Started ..." "Start")])
 
 (defn- project-delete-button
   [state]
-  [m/Button {:type "button"
+  [m/Button {:type     "button"
+             :theme    ["text-secondary-on-secondary-light"]
              :on-click #(reset! state true)} "Delete"])
 
+(defn- section-header
+  [number title]
+  [:div {:class-name "step-header"}
+   [:h2 [:span title]]])
+
 (defn edit-current-project
   []
   (let [current-project (subscribe [:projects2/current-project])
-        delete?         (r/atom false)]
+        delete? (r/atom false)]
     (fn []
       [ui/fixed-width (common2/nav-params)
        [ui/panel {}
         [m/Grid {}
          [m/GridCell {:span 6}
           [:form.vertical
-           [:h2 "Goal"]
-           [current-project-input "Goal" [:name] identity]
-           [regions-dropdown-component {:label "Region"
-                                        :on-change #(dispatch [:projects2/save-key :region-id %])
-                                        :model (:region-id @current-project)}]
-           [:h2 "Demand"]
-           [population-dropdown-component {:label "Sources"
-                                           :value (:population-source-id @current-project)
-                                           :on-change #(dispatch [:projects2/save-key :population-source-id %])}]
-           [current-project-input "Unit" [:config :demographics :unit-name] identity]
-           [current-project-input "Target" [:config :demographics :target] valid-input]
-           [m/TextFieldHelperText {:persistent true} (str "Percentage of population that should be considered " (get-in @current-project [:config :demographics :unit-name]))]
+           [:section {:class-name "project-settings-section"}
+            (section-header 1 "Goal")
```

Comment:
While in this case it's the exact same result, you should use the reagent convention of using `[]` when using child components. Ditto for the other uses of the `section-header` component.